### ISSUE TITLE
Icons missing alt text - SimpleReport Homepage

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4029]
## Area or Component
Home page - SimpleReport.gov

## Quick Summary
There are three expansion buttons on the home page missing alt text for their associated labels.

## Accessibility Violation
Text alternative is missing: https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-missing

## Action Requested
Add alt text to the three SVG images - the checkmark, laptop icon, and building icon

## Screenshots (optional)
![Screen Shot 2022-07-21 at 5 21 02 PM](https://user-images.githubusercontent.com/80282552/180336111-a31da6c0-6a3a-46bd-ad32-584d2756ba7a.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=3:5
